cartscript.js

```diff
@@ -0,0 +1,77 @@
+
+const cart=document.querySelector("cart")
+const cartRemove_btns= document.querySelectorAll(".trash-btn");
+addCartRemoveEvent();
```

Comment:
No need to extract the adding event listeners logic into functions that you just call right away. You can do it directly:

```js
const cartRemove_btns = document.querySelectorAll('.trash-btn');
cartRemove_btns.forEach(function(btn) {
  btn.addEventListener('click', handleRemoveCart);
});
```

or you can use an IIFE instead. Read more about JS IIFE [here](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).